python/dgl/data/gindt.py

```diff
@@ -42,19 +34,26 @@ class GINDataset(object):
         add self to self edge if true
     degree_as_nlabel: boolean
         take node degree as label and feature if true
-
+    Returns
+    -------
+    GINDataset object as an iterable for dataloader.
+    Examples
+    --------
+    >>> data = GINDataset(name='MUTAG', self_loop=False)
+    >>> data[0]
+    (Graph(num_nodes=23, num_edges=54,
+           ndata_schemes={'label': Scheme(shape=(), dtype=torch.int64), 'attr': Scheme(shape=(7,), dtype=torch.float64)}
+           edata_schemes={}), 0)
```

Comment:
does the example be
for g in data:
    g... ?

Reply:
Should we unify the example format in a later round of docstring update?